packages/server/scripts/database.sql

```diff
@@ -34,6 +35,36 @@ CREATE TABLE Event (
     ON DELETE SET NULL
 );
 
+CREATE TABLE user_guild (
+  user_id VARCHAR(255),
+  guild_id VARCHAR(255),
+  FOREIGN KEY(user_id)
+    REFERENCES users(user_id)
+    ON UPDATE CASCADE
+    ON DELETE SET NULL,
+  FOREIGN KEY(guild_id)
+    REFERENCES Guild(guild_id)
+    ON UPDATE CASCADE
+    ON DELETE SET NULL,
+  PRIMARY KEY(user_id, guild_id),
+  user_role VARCHAR(255),
+  points SMALLINT,
+  user_admin SMALLINT
```

Comment:
I think it's best to just name this `rank` instead. The higher the `rank` value the more privilege access to the Guild. For instance, a Guild member can post an event if they are above `100`.

Reply:
Can you clarify what column of this table to be renamed to rank? You selected 3 possible columns in the table in your comment.

Reply:
Can you clarify what column of this table needs to be renamed to rank? You selected 3 possible columns in the table in your comment.

Reply:
I was thinking of the `user_admin`.